packages/cli/src/config/PublishDetails.ts

```diff
@@ -371,14 +375,28 @@ const getAndroidDetails = async (
       );
     }
 
+    if (developerOverridenLocales == null && localeArray.length >= 60 || developerOverridenLocales.length > 60) {
```

Comment:
this will cause a runtime err if developerOverridenLocales is null and `localeArray.length < 60`, `developerOverridenLocales.length` will still be accessed, we can use `developerOverridenLocales?.length > 60`